src/main/java/core/service/FruitStore.java

```diff
@@ -0,0 +1,62 @@
+package core.service;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitStore {
+
+    public FruitStore() {
+
+    }
+
+    public List<OperationData> processOperations(List<OperationData> dataList) {
+        Map<String, Integer> fruitQuantityMap = new HashMap<>();
+
+        for (OperationData data : dataList) {
+            String product = data.getProduct();
+            int number = data.getNumber();
+            int currentQuantity = fruitQuantityMap.getOrDefault(product, 0);
+            int newQuantity = 0;
+
+            switch (data.getOperationType()) {
+                case "b":
+                case "s":
+                case "r":
+                    newQuantity = currentQuantity + number;
+                    break;
+                case "p":
+                    newQuantity = currentQuantity - number;
+                    break;
+                default:
+                    break;
+            }
```

Comment:
Do not use switch it is violation of Open Closed principle, use enum give them code value and check in for loop their code like this: 
for (Operation operation : Operation.values()) {
                if (operation.code.equals(operationLetter)) {
                    return operation;
                }
            }